scripts/UpdateMetaModule.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
```

Comment:
This script will be used more than just the metamodule. Can you rename it to UpdateModuleDependencies.ps1

Reply:
Done